src/helm/benchmark/run_specs.py

```diff
@@ -620,6 +620,15 @@ def get_cleva_machine_translation_metric_specs() -> List[MetricSpec]:
     ] + get_basic_metric_specs([])
 
 
+def get_cleva_paraphrase_generation_metric_specs() -> List[MetricSpec]:
```

Comment:
We should allow users to pass `alpha` outside to overwrite this coefficient and put a default 0.8 value here.